### ISSUE TITLE
FEATURE REQUEST: Ability to make pixel adjustments using arrow keys

### ISSUE THREAD

**Issue by tranquilcarbon** [Ross-Patterson/Portals-Desktop-Organization#214]
**Is your feature request related to a problem? Please describe.**
I have a lot of difficulty moving a portal to the exact location, i find my mouse moves several pixels at a time, sometimes.

**Describe the solution you'd like**
I think that the ability to make pixel adjustments to a portal's positioning would be good. Most applications that have something like this use the arrow keys, since they are easy to understand

**Describe alternatives you've considered**
I'm not sure about any other alternatives.

**Comment by rconstantine** [comment]
To do this would require a move mode or something. It would probably be easier to enable up/down arrows on the Portals app Home tab, where all the locations and dimensions are plainly visible. Enabling a mouse scroll wheel there would also be nice. If I could click in the height field and scroll, that would be awesome. Also, it would be nice if it kept whole numbers instead of decimals.

**Comment by Ross-Patterson** [comment]
Maybe in the cell next to the value I could have up/down arrows for height/Y and left/right arrows for width/X

This is a really rudimentary mockup, but something like this:
![image](https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/66630787/39328aa6-6542-4b95-b85f-aeb94c42d3ee)

**Comment by rconstantine** [comment]
Yeah, totally that. Scroll wheel enabled would be good though too.
